confuse/util.py

```diff
@@ -129,14 +129,20 @@ def find_package_path(name):
     return os.path.dirname(os.path.abspath(filepath))
 
 
+def xdg_config_home():
+    """Return the value of the XDG_CONFIG_HOME
+    environment variable if it is set and nonempty,
+    and UNIX_DIR_FALLBACK otherwise
+    """
+    config_path = os.getenv('XDG_CONFIG_HOME', UNIX_DIR_FALLBACK)
+    return config_path if config_path != '' else UNIX_DIR_FALLBACK
```

Comment:
Empty string is falsy. Use as a boolean.

```suggestion
    return config_path or UNIX_DIR_FALLBACK
```